q2_types/feature_data/tests/data/taxonomy/header-only.tsv

```diff
@@ -1,7 +1,7 @@
-# This file
 
-# only has a
-# header!
+
```

Comment:
No need to keep all these blank lines, we have another test file specifically for this.